src/docs/get-started/codelab.md

```diff
@@ -326,40 +331,44 @@ a child inside the existing `MyApp` stateless widget.
  3. Add the `build()` method to `RandomWordsState`:
 
     <!-- skip -->
+    <?code-excerpt "3-stateful-widget/lib/main.dart (RandomWordsState)" indent-by="2" replace="/(\n  )(.*)/$1[[highlight]]$2[[\/highlight]]/g"?>
     {% prettify dart %}
       class RandomWordsState extends State<RandomWords> {
         [[highlight]]@override[[/highlight]]
         [[highlight]]Widget build(BuildContext context) {[[/highlight]]
-          [[highlight]]final wordPair = WordPair.random();[[/highlight]]
-          [[highlight]]return Text(wordPair.asPascalCase);[[/highlight]]
+        [[highlight]]  final wordPair = WordPair.random();[[/highlight]]
+        [[highlight]]  return Text(wordPair.asPascalCase);[[/highlight]]
         [[highlight]]}[[/highlight]]
       }
     {% endprettify %}
 
- 4. Remove the word generation code from `MyApp`:
+ 4. Remove the word generation code from `MyApp` by making the changes shown in
+    the following diff:
 
     <!-- skip -->
-    {% prettify dart %}
-      class MyApp extends StatelessWidget {
-        @override
-        Widget build(BuildContext context) {
-          [[strike]]final wordPair = WordPair.random();[[/strike]]  // Delete this line
-
-          return MaterialApp(
-            title: 'Welcome to Flutter',
-            home: Scaffold(
-              appBar: AppBar(
-                title: Text('Welcome to Flutter'),
-              ),
-              body: Center(
-                //child: [[highlight]]Text(wordPair.asPascalCase),[[/highlight]] // Change the highlighted text to...
-                child: [[highlight]]RandomWords(),[[/highlight]] // ... this highlighted text
-              ),
-            ),
-          );
-        }
-      }
-    {% endprettify %}
+    <?code-excerpt "2-use-package/lib/main.dart" diff-with="3-stateful-widget/lib/main.dart" to="}"?>
+    ```diff
```

Comment:
Without diff formatting, this looks worse than the currently published version. Would it be possible to add the diff formatting in this PR?

Reply:
Pretty diffs are in the next PR, which is almost done:

<img width="621" alt="screen shot 2018-12-13 at 13 04 48" src="https://user-images.githubusercontent.com/4140793/49960166-97a1f000-fedd-11e8-822c-f3bd91e2105f.png">